pages/mesosphere/dcos/1.14/installing/production/advanced-configuration/configuration-reference/index.md

```diff
@@ -248,6 +250,13 @@ Use this to set up a custom CA certificate. See [using a Custom CA Certificate](
 
 Use this to set up a custom CA certificate. See [using a Custom CA Certificate](/mesosphere/dcos/1.14/security/ent/tls-ssl/ca-custom#configuration-parameter-reference) page for a detailed configuration parameter reference. 
 
+### exhibitor_tls_required [enterprise type="inline" size="small" /]
+By default this option is set to `false`, when `true` failures during Exhibitor TLS initialization are treated critical exceptions. Otherwise, failures are logged and Exhibitor continues to operate without TLS protections.
+
+### exhibitor_bootstrap_ca_url [enterprise type="inline" size="small" /]
+
+Optional parameter which specifies the bootstrap CA service URL. If not present, the URL is determined using the bootstrap_url.
```

Comment:
... used for generating the TLS artifacts for the automated Exhibitor lock-down.